samples/nrf9160/modem_shell/prj.conf

```diff
@@ -118,6 +119,10 @@ CONFIG_DATE_TIME_UPDATE_INTERVAL_SECONDS=0
 CONFIG_NRF_CLOUD_AGPS=y
 CONFIG_NRF_CLOUD_REST=y
 CONFIG_MODEM_JWT=y
+# Uncomment the following if CONFIG_MOSH_CLOUD is enabled
```

Comment:
Makes me wonder if there should be yet another overlay for MQTT. :)

Reply:
Actually I found out that you don't even need to set CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD=y and CONFIG_MQTT_KEEPALIVE=1200 as they are already default values so I think we don't need an another overlay quite yet :)